packages/@aws-cdk/core/lib/duration.ts

```diff
@@ -143,6 +143,36 @@ export class Duration {
     }
   }
 
+  /**
+   * Turn this duration into a human-readable string
+   */
+  public toHumanString(): string {
+    if (this.amount === 0) { return fmtUnit(0, this.unit); }
+    if (Token.isUnresolved(this.amount)) { return `<token> ${this.unit.label}`; }
```

Comment:
Why not do `${this.amount} {this.unit}` instead? The token should get resolved, yes?
It might not be the best output and we might end up something like '458321 seconds' but at least it's something.

Reply:
Oh, nevermind. I realized we can't do string operations on numeric tokens.

Reply:
Would it be better to error instead?

Reply:
Do you think that's reasonable? I feel it's more reasonable to render what we got, which is "<unknown> minutes", and let the human decide whether that's good enough for them or not.

If not, they can always add the check themselves.